gradle.properties

```diff
@@ -21,3 +21,6 @@ jdkVersion = 8
 
 # defaultScalaVersion is used to specify the version of Scala to build and test Gravitino
 defaultScalaVersion = 2.12
+
+# sparkMajorVersion is used to specify the major version of Spark to build and test Gravitino spark connector
+sparkMajorVersion = 3.4
```

Comment:
SparkMajorVersion should be calculated from SparkVersion.

Reply:
done